src/components/AccountSettings/AccountSettings.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
Do we need this file anymore?  Or should it be changed?

Reply:
Removed this file